### DISCUSSION TITLE
Does Dual/Quad Channel RAM improves plotter speed?

### DISCUSSION THREAD

**Discussion by MaximusChia** [Chia-Network/chia-blockchain#5352] [General]
I have a DDR 4 Kingston Hyper X Predator, since 16GB just allow me to plot 4 in parallel, I plan to add an extra module, and take advantage of Dual channel.
Will this cause noticeable speed up on my plotting performance?

**Comment by Zerophase** [comment]
I don't know for sure. But, it should since you can write and read more memory at once. I think memory access should be faster. Of course, this depends on your bottleneck.
